src/appendices/installing-software/setting-up-vsc.rst

```diff
@@ -0,0 +1,45 @@
+.. _vsc-install:
+
+Setting Up Visual Studio Code
+=============================
+
+.. index:: ! IDE, ! integrated development environment
+
+Before we start coding on our computer, we need to make sure we have the right tools! Programmers use **integrated development environments** (IDE) to write and run their code.
+The development environment we will be using for this class is Visual Studio Code.
+In addition to simply writing and running code, Visual Studio Code has tools that recognize errors in our code and has an integrated terminal so we can navigate through our filesystem to find the files that need our attention.
+
+Mac Users
+---------
+
+Go to the Visual Studio Code `download page <https://code.visualstudio.com/download/>`_ and download Visual Studio Code for Mac.
+
+.. note::
+
+   Visual Studio Code is very customizable. Once you have everything set up, you can take additional steps to personalize your workspace such as changing the theme.
+
+
+Getting Started
+^^^^^^^^^^^^^^^
+
+Open your new copy of Visual Studio Code. To open one of your coding projects, go to `File > Open` and select the project you want to work on.
+
+To start working with the terminal, go to `Terminal > New Terminal`. The new terminal window will open on the bottom right.
+
+Windows Users
+-------------
+
+Go to the Visual Studio Code `download page <https://code.visualstudio.com/download/>`_ and download Visual Studio Code for Mac.
+
+Getting Started
+^^^^^^^^^^^^^^^
+
+Open your new copy of Visual Studio Code. To open one of your coding projects, go to `File > Open` and select the project you want to work on.
+
+To start working with the terminal, go to `Terminal > New Terminal`. The new terminal window will open on the bottom right.
+
+However, the terminal is in powershell, not Git Bash. To change this, open the Command Palette by going to `View > Command Palette`. Type "Select Default Shell" in the search window and select "Select Default Shell" from the menu.
```

Comment:
What do you think about adding a heading named "Using Git Bash"?

Reply:
Reworked this page!